app/src/main/java/com/alfresco/content/app/activity/LoginActivity.kt

```diff
@@ -51,7 +52,7 @@ class LoginActivity : com.alfresco.auth.activity.LoginActivity() {
                 authConfig.jsonSerialize(),
                 endpoint,
                 person.displayName ?: "",
-                person.email,
+                person.email ?: "",
```

Comment:
Email is mandatory so no need to pass default value

Reply:
done